tests/bcr/.bazelversion

```diff
@@ -1 +1 @@
-6.0.0
+6.3.2
```

Comment:
Just curious, why is this bump necessary?

Reply:
whoops, that's not necessary. reverting